addon/components/flexberry-map.js

```diff
@@ -77,6 +90,7 @@ export default Ember.Component.extend(
       this.set('_layer', map);
       this._addObservers();
       this._addEventListeners();
+      this.sendAction('leafletMapDidInit', map);
```

Comment:
It raise after creating L.Map instance. Yes, its ugly but describe exactly what happen